otx/algorithms/common/adapters/mmcv/models/backbones/efficientnet.py

```diff
@@ -15,14 +15,15 @@
 import os
 
 import torch.nn.functional as F
-from mmcls.models.builder import BACKBONES
 from mmcv.cnn import build_activation_layer
 from mmcv.cnn.bricks import ConvModule
 from mmcv.runner import load_checkpoint
 from mpa.utils.logger import get_logger
 from torch import nn
 from torch.nn import init
 
+from ..builder import BACKBONES
```

Comment:
I saw this BACKBONES is from registry MMCV_MODELS. Is it compatible with mmcls backbone? I mean did it find well as classification backbones? or these models are used globally across the tasks?

Reply:
The backbone of mmcls also uses the backbone of mmcv. It is registered as one registry and configured to be used at once.

The backbones of mmcls need some more checking, but first my goal is to register all backbones in the registry under mmcv.